lms/envs/devstack.py

```diff
@@ -117,15 +117,36 @@ def should_show_debug_toolbar(_):
 ########################### VERIFIED CERTIFICATES #################################
 
 FEATURES['AUTOMATIC_VERIFY_STUDENT_IDENTITY_FOR_TESTING'] = True
-FEATURES['ENABLE_PAYMENT_FAKE'] = True
-
-CC_PROCESSOR_NAME = 'CyberSource2'
+FEATURES['ENABLE_PAYMENT_FAKE'] = False
+
+# CC_PROCESSOR_NAME = 'CyberSource2'
```

Comment:
Why do we need these commented lines?

Reply:
I forgot to delete these lines. I deleted it. (I forgot to delete these lines)